src/app/static/src/app/components/Stepper.vue

```diff
@@ -130,6 +132,24 @@
             SurveyAndProgram,
             LoadingSpinner,
             ModelRun
+        },
+        watch: {
+            ready: function (newVal) {
+                if (newVal) {
+                    const activeStep = localStorageManager.getInt("activeStep");
+
+                    if (activeStep) {
+                        const invalidSteps = [1, 2, 3, 4, 5]
+                            .filter((i) => i < activeStep && !this.complete[i]);
```

Comment:
Could use Object.keys(this.complete) here to avoid the hardcoded array which assumes step number, though in that case you'd get them back as strings I think.

Reply:
realised could do `this.steps.map((s: Step) => s.number)` which seems nicer